tests/webpack.config.error.js

```diff
@@ -10,6 +9,7 @@ const crypto_orig_createHash = crypto.createHash;
 crypto.createHash = algorithm => crypto_orig_createHash(algorithm == "md4" ? "sha256" : algorithm);
 
 module.exports = {
+  mode: "none",
```

Comment:
Without setting mode, we get a very noise warning about how it's important to set mode on each test